# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/moz.build
# Commit: 35dde62c945a
# Full Hash: 35dde62c945ab7f66f6fd48a8b4cf706c3f78fb2
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 12: Redirect EarlyHint preload channel to content process r=necko-reviewers,kershaw,valentin
#   
#   Similar to how DocumentLoadListener redirects the document to the content process.
#   
#   This fixes the race conditions in early hint architecture fixing intermittent
# ==============================================================================

diff -r ca2681b0f792 -r 35dde62c945a netwerk/ipc/moz.build
--- a/netwerk/ipc/moz.build	Fri Dec 02 16:15:57 2022 +0000
+++ b/netwerk/ipc/moz.build	Fri Dec 02 16:15:57 2022 +0000
@@ -41,6 +41,7 @@
     "InputChannelThrottleQueueChild.cpp",
     "InputChannelThrottleQueueParent.cpp",
     "NeckoChild.cpp",
+    "NeckoCommon.cpp",
     "NeckoParent.cpp",
     "NeckoTargetHolder.cpp",
     "ParentChannelWrapper.cpp",